src/TeamSpeak3QueryApi/Interfaces/ITypeCaster.cs

```diff
@@ -2,8 +2,9 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using TeamSpeak3QueryApi.Net.Extensions;
 
-namespace TeamSpeak3QueryApi.Net.Specialized
+namespace TeamSpeak3QueryApi.Net.Interfaces
```

Comment:
I think "interfaces" is not a good name for a namespace. We should come up with something more descriptive.

Reply:
I couldn't change this to just interface cause we have a interface class... Any ideas?